packages/core/src/components/button/_common.scss

```diff
@@ -239,6 +263,10 @@ $button-intents: (
   background-image: none;
 }
 
+@mixin pt-dark-button-placeholder() {
```

Comment:
not convinced these mixins are helpful, since they just apply a single variable. please remove.

Reply:
sounds good -- pattern-matched too closely.